.github/workflows/workflow_inference.yml

```diff
@@ -45,6 +45,8 @@ jobs:
           - { model: "gpt-j-6b"}
           - { model: "mistral-7b-v0.1"}
           - { model: "mpt-7b-bigdl"}
+          - { model: "CodeLlama-7b-hf"}
+          - { model: "falcon-7b"}
```

Comment:
please review them before merging it since we've verified them successfully.

Reply:
Ok, I will remove it as soon as it's ready to merge.

Reply:
Removed.